### PR TITLE
Add secondary manifest to okbuck_manifest rule

### PR DESCRIPTION
**Pull request by raviagarwal7**
This fixes manifest genrule to generate correct merged manifest for flavors.

### CODE REVIEW

File: buildSrc/src/main/java/com/uber/okbuck/composer/jvm/JvmLibraryRuleComposer.java [hunk 1]
```diff
@@ -10,6 +10,7 @@
 import java.util.List;
 import java.util.Set;
 
+@SuppressWarnings("NullAway")
```
Comment (kageiit) [inline buildSrc/src/main/java/com/uber/okbuck/composer/jvm/JvmLibraryRuleComposer.java]: Why is this related?

Comment (raviagarwal7) [reply buildSrc/src/main/java/com/uber/okbuck/composer/jvm/JvmLibraryRuleComposer.java]: not sure, nullaway threw errors on
```
          new JvmBinaryRule()
              .mainClassName(target.getMainClass())
```
target.getMainClass() could be null

not sure why it isn't wasn't caught before
